src/Dhadgar.Scope/src/components/layout/MainLayout.tsx

```
@@ -1,54 +1,54 @@
+      <div className="mb-4 flex items-center justify-between rounded-2xl border border-white/10 bg-white/5 p-4 lg:hidden">
+        <a href="/" className="block">
+          <div className="text-sm text-white/60">Meridian Console</div>
+          <div className="text-xl font-bold tracking-tight">Scope</div>
+        </a>
+
+        <button
+          type="button"
+          onClick={() => setSidebarOpen(true)}
+          className="rounded-lg border border-white/10 bg-white/5 p-2 transition-colors hover:bg-white/10"
+        >
+          <svg className="h-5 w-5" fill="none" stroke="currentColor" viewBox="0 0 24 24">
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)